### DISCUSSION TITLE
A way to get the same information as the original "Following" page

### DISCUSSION THREAD

**Discussion by yan12125** [community/community#37484] [Other Feature Feedback, Questions, & Ideas]
Update: better alternatives:
* https://github.com/community/community/discussions/37693
* https://github.com/community/community/discussions/37694

---

Apparently there are already plenty of complaints. Let me share something useful.

I didn't find a way to get the the original "Following" page back to the homepage, but there is a way to get the same information:

1. Create a [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
2. Use [the feed API](https://docs.github.com/en/rest/activity/feeds#get-feeds) with the personal access token to get available feed URLs
3. In API results, you will see a line like
```
  "current_user_url": "https://github.com/<username>.private.atom?token=xxx",
```
This URL is an RSS feed. It contains the same entries as the original "Following" page. Put it in your favorite RSS reader and forget about the new useless homepage :)

**Comment by Harsh-br0** [comment]
This seems useful and thanks for such suggestion. It would be good if frontend devs able to design a web app too by utilising this feed.

**Reply by vi** [reply]
I don't see `current_user_url` in the reply, and the only `private.atom?token=xxx` URLs in the reply start with `https://github.com/organizations/`

**Reply by yan12125** [reply]
Did you have `._links.current_user` in the reply? It has the same URL as `current_user_url`
```
  "_links": {
    ...
    "current_user": {
      "href": "https://github.com/<username>.private.atom?token=xxx",
      "type": "application/atom+xml"
    },
    ...
  }
```

**Reply by bater53** [reply]
Since I am new to Github.  Can anyone explain to me how do we get updates or information on any changes that may occur?  It seems like alot of this is at least 6-9 months old and I see no updates or really much response.  So how does one know if any of this is relevant?  Thank you for your reccommendations in advance.

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
